Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+# Modified from Configuration/Generator/python/TenMuE_0_200_pythia8_cfi.py
+generator = cms.EDFilter("Pythia8EGun",
```

Comment:
Ciao Mia, isn't "Pythia8EGun" the same name used for the ten muon gun EDFilter? Wouldn't it be better to change the name here?